src/styles/blocks/card.scss

```diff
@@ -0,0 +1,113 @@
+.card {
+  width: $card-width;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  font-family: Roboto, serif;
+  font-weight: 400;
+
+  transition: all linear $main-during;
```

Comment:
Ensure that the SCSS variable `$main-during` is defined. This variable is crucial for the transition duration.